pkg/download/downloader.go

```diff
@@ -72,6 +74,19 @@ func extractZIP(targetDir string, read io.ReaderAt, size int64) error {
 			return errors.Wrap(err, "could not open inflating zip file")
 		}
 
+		if f.FileInfo().Mode()&os.ModeSymlink == os.ModeSymlink {
+			buf := new(bytes.Buffer)
+			_, err := io.Copy(buf, src)
+			if err != nil {
+				return errors.Wrap(err, "problem reading symlink target")
+			}
+			oldname := strings.TrimSpace(buf.String())
```

Comment:
why trimming space? maybe the filename has a trailing space that requires preserving?